### PR TITLE
Added Neighbors support and a 404 page.

### PR DESCRIPTION
**Pull request by bnice5000**
This is a rework of PR #397.

### CODE REVIEW

File: pelican-bootstrap3/README.md [hunk 1]
```diff
@@ -1,9 +1,15 @@
 # pelican-bootstrap3
 
+<<<<<<< HEAD
```
Comment (justinmayer) [inline pelican-bootstrap3/README.md]: Diff info here should be removed, yes?

Comment (bnice5000) [inline pelican-bootstrap3/README.md]: Yes that is correct.

**Comment by justinmayer** [comment]
Thanks for the contribution, Brian. Let me know once the extraneous diff info has been removed and the commit has been rebased/squashed. Thanks!

**Comment by almet** [comment]
Hi, sorry for the delay on this, could you consider fixing the problematic line so we can merge this? Thanks!

**Comment by justinmayer** [comment]
Hi Brian. Care to remove the spurious extra lines from the ReadMe so we can get this merged?

**Pull request closed without merging**